tests/test_jobs.py

```diff
@@ -0,0 +1,293 @@
+
+        job = enqueue(job_echo, 'ABC')
+        qjob = find_job(job.id)
+        self.assertEqual(qjob, job)
+
+    def test_not_found_job(self):
+        """Check if it raises an exception when the job is not found"""
+
+        enqueue(job_echo, 'ABC')
+
+        with self.assertRaisesRegex(NotFoundError,
+                                    JOB_NOT_FOUND_ERROR):
+            find_job('DEF')
+
+
+class TestAffiliateIndividuals(TestCase):
+    """Unit tests for affiliate"""
+
+    def setUp(self):
+        """Initialize database with a dataset"""
+
+        self.user = get_user_model().objects.create(username='test')
+        ctx = SortingHatContext(self.user)
+
+        # Organizations and domains
+        api.add_organization(ctx, 'Example')
+        api.add_domain(ctx, 'Example', 'example.com', is_top_domain=True)
+
+        api.add_organization(ctx, 'Example Int.')
+        api.add_domain(ctx, 'Example Int.', 'u.example.com',
+                       is_top_domain=True)
+        api.add_domain(ctx, 'Example Int.', 'es.u.example.com')
+        api.add_domain(ctx, 'Example Int.', 'en.u.example.com')
+
+        api.add_organization(ctx, 'Bitergia')
+        api.add_domain(ctx, 'Bitergia', 'bitergia.com')
+        api.add_domain(ctx, 'Bitergia', 'bitergia.org')
+
+        api.add_organization(ctx, 'LibreSoft')
+
+        # John Smith identity
+        self.jsmith = api.add_identity(ctx,
+                                       source='scm',
+                                       email='jsmith@us.example.com',
+                                       name='John Smith',
+                                       username='jsmith')
+        api.add_identity(ctx,
+                         source='scm',
+                         email='jsmith@example.net',
+                         name='John Smith',
+                         uuid=self.jsmith.uuid)
+
+        # Add John Doe identity
+        self.jdoe = api.add_identity(ctx,
+                                     source='unknown',
+                                     email=None,
+                                     name='John Doe',
+                                     username='jdoe')
+
+        # Jane Roe identity
+        self.jroe = api.add_identity(ctx,
+                                     source='scm',
+                                     email='jroe@example.com',
+                                     name='Jane Roe',
+                                     username='jroe')
+        api.add_identity(ctx,
+                         source='scm',
+                         email='jroe@example.com',
+                         uuid=self.jroe.uuid)
+        api.add_identity(ctx,
+                         source='unknown',
+                         email='jroe@bitergia.com',
+                         uuid=self.jroe.uuid)
+
+    def test_affiliate(self):
+        """Check if all the individuals stored in the registry are affiliated"""
+
+        ctx = SortingHatContext(self.user)
```

Comment:
I have a question here: It is necessary to create a new context before performing each request? Could the context defined in the `setUp` be used for all the requests?

Reply:
Probably it's not necessary but I wanted to be user I was using a new context each time.